app/views/static_pages/home/_success_stories.html.erb

```diff
@@ -31,9 +31,18 @@
     <%= link_to 'Read more success stories', success_stories_path, class: 'secondary' %>
   </div> <!-- /.success-stories-read-more -->
 
-  <%= render 'shared/bottom_cta', 
-      button: sign_up_button,
-      heading: 'Start learning for free now!',
-      subheading: ''
-  %>
+  <% unless current_user %>
```

Comment:
with your new helper method don't need this logic 😄 
just change the button value in the render to call your helper method and it will display the correct button here:
```
 <%= render 'shared/bottom_cta', 
          button: your_helper_method_here,
          heading: 'Start learning for free now!',
          subheading: ''
 %>
```